packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactRootView.java

```diff
@@ -434,10 +434,8 @@ public void run() {
 
     if (mShouldLogContentAppeared) {
       mShouldLogContentAppeared = false;
-
-      if (mJSModuleName != null) {
-        ReactMarker.logMarker(ReactMarkerConstants.CONTENT_APPEARED, mJSModuleName, mRootViewTag);
-      }
+      String jsModuleName = getJSModuleName();
```

Comment:
> could help us catching the issue as early as possible

I agree with this when adding a new API, but modifying an API to introduce this, if the previous behavior wasn't crashing, is not ideal.

Given that we already have some callers in the critical path (e.g.: `startSurface`) that were using the getter with the assert, I think it might be safe to continue using it, even for this use case.

Reply:
Sorry I think you misunderstood. I meant that you can keep the implementation you had replacing all usages with `getJSModuleName` with the assert because we were already calling that when loading a new surface. If there were cases where it was defined, that should've thrown an error in those cases too. So replacing all with `getJSModuleName` and removing the null checks should be fine.